src/pages/Join/phoneAuthPage.tsx

```diff
@@ -25,24 +28,34 @@ export default function PhoneAuthPage() {
     defaultValues: { phoneNumber: '', isAuthed: false },
   });
   const navigate = useNavigate();
-  const [isPhoneDuplicated, isAuthed] = watch([
+  const [authNumber, isPhoneDuplicated, isAuthed] = watch([
+    'authNumber',
     'isReceivedAuthNum',
     'isAuthed',
   ]);
 
+  const [authNum, setAuthNum] = useState(0);
+  const { data, isSuccess } = useAuthNumQuery(authNum);
+
   const sendAuthNum = () => {
     if (isPhoneDuplicated) {
+      setAuthNum(+authNumber); // TODO: authNumber이 number 타입인데 string으로 들어감..
```

Comment:
이거 혹시 타입 단언문 쓸 수 있나요? 단언이 가능하면 좀 더 직관적일 것 같아서요!
```suggestion
      setAuthNum(authNumber as number); // TODO: authNumber이 number 타입인데 string으로 들어감..
```